SdkVersion.props

```diff
@@ -3,20 +3,25 @@
 <Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
     <SDKVersionRS2>10.0.15063.0</SDKVersionRS2>
-	<SDKVersionRS3>10.0.16299.0</SDKVersionRS3>
-	<SDKVersionRS4>10.0.17134.0</SDKVersionRS4>
-	<SDKVersionRS5>10.0.17763.0</SDKVersionRS5>
-	<SDKVersion19H1>10.0.18362.0</SDKVersion19H1>
-	<SDKVersionInsider>10.0.18362.0</SDKVersionInsider>
+    <SDKVersionRS3>10.0.16299.0</SDKVersionRS3>
+    <SDKVersionRS4>10.0.17134.0</SDKVersionRS4>
+    <SDKVersionRS5>10.0.17763.0</SDKVersionRS5>
+    <SDKVersion19H1>10.0.18362.0</SDKVersion19H1>
+    <SDKVersionInsider>10.0.18362.0</SDKVersionInsider>
```

Comment:
>SDKVersionInsider [](start = 5, length = 17)

I would suggest to reuse SDKVersionInsider other than creating new SDKVersionInternal

Reply:
+1, this SDK will be the insider one soon.

Reply:
Switched back to using UseInsiderSDK and SDKVersionInsider.